homeassistant/components/zwave/workaround.py

```diff
@@ -68,28 +68,19 @@
 
 def get_device_component_mapping(value):
     """Get mapping of value to another component."""
-    if (value.node.manufacturer_id.strip() and
-            value.node.product_type.strip()):
```

Comment:
Empty string is considered false:

![image](https://cloud.githubusercontent.com/assets/1444314/24283540/aea2dcf6-1023-11e7-82c2-b8f2745958e7.png)

But I don't know if manufacturer_id or product_type would ever return a string that is just whitespace.